charts/renku/requirements.yaml

```diff
@@ -2,23 +2,23 @@ dependencies:
 - name: renku-ui
   alias: ui
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.5.1"
+  version: "0.6.2"
 - name: renku-notebooks
   alias: notebooks
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.3.3"
+  version: "0.4.0"
 - name: renku-gateway
   alias: gateway
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.4.0"
+  version: "0.4.1"
 - name: gitlab
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
   version: "0.3.1-6fb1535"
   condition: gitlab.enabled
 - name: renku-graph
   alias: graph
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.5.0"
+  version: "0.6.0-5411305"
```

Comment:
Is this intentional or by accident?

Reply:
intentional - though it's not the latest version atm